website/src/pages/api/v0/settings/flags.ts

```diff
@@ -11,8 +11,9 @@ export default async function handle (req: NextApiRequest, res: NextApiResponse<
   await runMiddleware(req, res, cors)
   res.json([{
     id: 'loan',
-    name: 'Decentralized Loans',
-    stage: 'alpha',
-    version: '>=0.12.0'
+    name: 'Loans',
+    stage: 'beta',
```

Comment:
@fullstackninja864 this should be Alpha, don't change it to Beta yet.

Reply:
Sure, will update it